packages/api/src/controllers/reviews/v1/mod.rs

```diff
@@ -32,19 +35,21 @@ impl ReviewControllerV1 {
         State(ctrl): State<ReviewControllerV1>,
         Path(id): Path<i64>,
     ) -> Result<Json<Review>> {
-        //TODO: implement get review
         let _repo = ctrl.review_repo;
         tracing::debug!("get_review: {:?}", id);
 
-        Ok(Json(Review::default()))
+        let fetched_review = _repo
+            .find_one(&ReviewReadAction::new().find_by_id(id))
+            .await?;
```

Comment:
https://github.com/biyard/dagit/pull/12/files#diff-343a52e8daaa2709929f84898ee3224e592c88a93dd2d765bc1705b21e01c442R83

I think it would be more intuitive to use `id_equals` as follows rather than using `find_by_id`.